source/form.html

```diff
@@ -0,0 +1,146 @@
+          <path opacity=".2" d="M20.188 6.857h-3.55L13.188.381C12.985 0 12.58-.095 12.174.095s-.507.572-.304.953l3.145 5.81H5.783l3.145-5.81c.202-.381 0-.762-.305-.953-.406-.19-.811 0-1.014.286l-3.45 6.476H.812c-.406 0-.812.286-.812.762 0 .381.304.762.812.762h.202l.914 2.857c-.203.095-.305.286-.305.572 0 .38.305.666.71.666L4.058 18h12.884l1.725-5.524c.405 0 .71-.286.71-.667 0-.19-.102-.476-.304-.57l.912-2.858h.203c.406 0 .812-.286.812-.762 0-.476-.406-.762-.812-.762Zm-4.565 9.62H5.073L3.854 12.38h13.087l-1.319 4.095Zm1.623-5.525H3.348l-.812-2.666h15.522l-.812 2.666Z" fill="#000"/>
+        </svg>
+        Корзина: пока пуста
+      </a>
+    </nav>
+  </header>
+  <main class="page-main">
+    <h1 class="main-title">Вязание на заказ</h1>
+    <form class="form" action="https://echo.htmlacademy.ru/" method="get">
+      <p class="form__description">
+        Мы будем рады воплотить в жизнь ваши пожелания! Заполните простую форму заказа и мы свяжемся с
+        вами, чтобы уточнить детали.
+      </p>
+      <fieldset class="form__group">
+        <legend class="form__legend">Тип</legend>
+        <input class="form__input form__input--radio" type="radio" name="accessory"
+        id="accessory" value="accessory" checked>
+        <label class="form__label" for="accessory">Аксессуар для интерьера</label>
+        <input class="form__input form__input--radio" type="radio" name="toy"
+        id="toy" value="toy">
+        <label class="form__label" for="toy">Детская игрушка</label>
+      </fieldset>
+      <fieldset class="form__group">
+        <legend class="form__legend">Цвет</legend>
+        <input class="form__input form__input--checkbox" type="checkbox" name="white"
+        id="white-color" checked>
+        <label class="form__label" for="white-color">Белый</label>
+        <input class="form__input form__input--checkbox" type="checkbox" name="gray"
+        id="gray-color" checked>
+        <label class="form__label" for="gray-color">Серый</label>
+        <input class="form__input form__input--checkbox" type="checkbox" name="tiffany"
+        id="tiffany-color">
+        <label class="form__label" for="tiffany-color">«Тиффани»</label>
+        <input class="form__input form__input--checkbox" type="checkbox" name="black"
+        id="black-color">
+        <label class="form__label" for="black-color">Черный</label>
+        <input class="form__input form__input--checkbox" type="checkbox" name="pink"
+        id="pink-color">
+        <label class="form__label" for="pink-color">Розовый</label>
+        <input class="form__input form__input--checkbox" type="checkbox" name="orange"
+        id="orange-color">
+        <label class="form__label" for="orange-color">Оранжевый</label>
+      </fieldset>
+      <fieldset class="form__group">
+        <legend class="form__legend">ФИО</legend>
+        <label class="form__label" for="name">Имя:</label>
+        <input class="form__input form__input--line" type="text" name="name" id="name"
```

Comment:
```suggestion
        <input class="form__input" type="text" name="name" id="name"
```

интересно, а за что у тебя модификатор (мод) отвечал?

Reply:
аналогично ниже - это стандартные инпуты так выглядят